### PR TITLE
Clean alerts

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: src/GearsBuilder.py [hunk 1]
```diff
@@ -1,10 +1,6 @@
 import redisgears
 import copy
-import redisgears as rg
 from redisgears import executeCommand as execute
-from redisgears import atomicCtx as atomic
```
Comment (MeirShpilraien) [inline src/GearsBuilder.py:5]: So you do not want our users to have atomic and hashtag functions?

File: recipes/write_behind/write_behind.py [hunk 2]
```diff
@@ -3,7 +3,6 @@
 VERSION = '99.99.99'
 NAME = 'WriteBehind'
 
-import time
```
Comment (MeirShpilraien) [inline recipes/write_behind/write_behind.py:6]: This entire file need to be deleted anyway...

Comment (gkorland) [reply recipes/write_behind/write_behind.py:6]: Is that the effect? I was just removing unused imports

Comment (MeirShpilraien) [reply recipes/write_behind/write_behind.py:6]: Its not unused, its unused in this file, do not believe those static analyzers....

**Comment by MeirShpilraien** [comment]
@gkorland can I close this?

**Comment by MeirShpilraien** [comment]
@gkorland ^^

**Pull request closed without merging**